pokedex/src/components/Router.js

```diff
@@ -1,19 +1,23 @@
 import React from "react";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter, Route, Routes, Navigate } from "react-router-dom";
 import App from "../App";
 import Detail from "./Detail";
-import Test from "./Test";
+import Invalid from "./Invalid";
+import Page from "./Page";
 
 const Router = () => {
-  return(
-  <BrowserRouter>
-    <Routes>
-      <Route path="/" element={<App />} />
-      <Route path="/pokemon" element={<App />} />
-      <Route path="/pokemon/:page_id" element={<Detail />} />
-      <Route element={<Test />} />
-    </Routes>
-  </BrowserRouter>);
-}
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Page />}>
+          <Route index element={<Navigate replace to={"/pokemon/"} />}></Route>
+          <Route path="/pokemon/" element={<App />} />
+          <Route path="/pokemon/:page_id" element={<Detail />} />
+        </Route>
+        <Route path="*" element={<Invalid />} />
+      </Routes>
+    </BrowserRouter>
+  );
+};
```

Comment:
The naming of this file is a bit odd, I would expect for `App.js` to be the top level file that encompasses the entire project and defines the routing structure, then the file that is currently called `App.js` might be something like `Home.js` as it is more specific than the entire app.